### ISSUE TITLE
Regression: Model store created with Keras models breaks with the latest merge

### ISSUE THREAD

**Issue by cdknorow** [operatorai/modelstore#144]
@nlathia  it looks like the change made to merge the Keras and TensorFlow managers may have broken older model stores. This is the error I see when loading a model that was previously saved.
```

def get_manager(name: str, storage: CloudStorage = None) -> ModelManager:
       manager = _LIBRARIES[name](storage)
E       KeyError: 'keras'
```

This actually brings up a good regression test.

Create a model store that has one of each type of model, then the test can try and load all the models to validate they are still working after any changes.

**Comment by nlathia** [comment]
Thanks! This one is a fairly straightforward fix, leave it with me.

**Comment by nlathia** [comment]
Good idea for the regression test too, I had started thinking about this sort of thing in relation to the windows path bug as well:

- https://github.com/operatorai/modelstore/issues/140

**Comment by nlathia** [comment]
@cdknorow PR with a fix is open! Feedback welcome. I'm going to set up a suite of more generic regression tests in a separate PR.

**Comment by nlathia** [comment]
This one is merged now and it will go out in the next release!

- https://github.com/operatorai/modelstore/pull/145

**Comment by nlathia** [comment]
Closing now, this has been released with `modelstore==0.0.74`

- https://github.com/operatorai/modelstore/pull/155
